### PR TITLE
Fixing ball radius and resolution when used to represent a 3D point 

### PR DESCRIPTION
**Pull request by dcoeurjo**
Static constant in Display3DFactory to specify the radius and resolution of a ball when used to display a 3D point. (see #965)

### CODE REVIEW

File: src/DGtal/io/viewers/Viewer3D.h [hunk 1]
```diff
@@ -223,7 +223,7 @@ namespace DGtal
     }
 
 
-    /// the 3 possible axes for the image direction
+    /// the 3 possible axes for the imaball3Dge direction
```
Comment (kerautret) [inline src/DGtal/io/viewers/Viewer3D.h]: something wrong with your editor ;)

**Comment by kerautret** [comment]
Looks fine now, just correct the small editor typo before merge ;)

**Comment by dcoeurjo** [comment]
done thanks

**Comment by kerautret** [comment]
ok thanks merging

**Pull request merged**